api/src/test/java/org/apache/iceberg/transforms/TestProjection.java

```diff
@@ -216,15 +219,15 @@ public void testCaseInsensitiveStrictIdentityProjection() {
 
   @Test
   public void testCaseSensitiveStrictIdentityProjection() {
-    PartitionSpec spec = PartitionSpec.builderFor(SCHEMA)
-      .identity("id")
-      .build();
+    PartitionSpec spec = PartitionSpec.builderFor(SCHEMA).identity("id").build();
 
     assertThrows(
-      "X != x when case sensitivity is on",
-      ValidationException.class,
-      "Cannot find field 'ID' in struct",
-      () -> { Projections.strict(spec, true).project(Expressions.notNull("ID")); });
+        "X != x when case sensitivity is on",
+        ValidationException.class,
+        "Cannot find field 'ID' in struct",
+        () -> {
+          Projections.strict(spec, true).project(Expressions.notNull("ID"));
+        });
```

Comment:
How about here? Manual change, or there is a rule for this?

Reply:
This one I believe is a rule where blocks inside `{}` have to start on a new line.

Reply:
Probably also should not be a block, right?